content/swift/concepts/dictionaries/terms/removeValue/removeValue.md

```diff
@@ -0,0 +1,43 @@
+---
+Title: '.removeValue()'
+Description: 'Removes a key-value pairing from a dictionary based on a given key.'
+Subjects:
+  - 'Mobile Development'
+  - 'Computer Science'
+Tags:
+  - 'Dictionaries'
+  - 'Properties'
+CatalogContent:
+  - 'learn-swift'
+  - 'paths/build-ios-apps-with-swiftui'
+---
+
+The **`.removeValue()`** method will remove a key-value pairing from a dictionary based on a given key. If the key exists in the dictionary, the method will return the removed value. However, if the key is not found, the method will return nil.
+
+## Syntax
+
+```pseudo
+dictionaryInstance.removeValue(forKey: key)
+```
+
+The **`dictionaryInstance`** is a dictionary that you want to remove a key-value pairing from. The **`key`** paramater is the key you would like to remove.
+
+## Example
+
```

Comment:
```suggestion

In the example below, the `fruitBasket` dictionary is checked to see if the key `Apples` exists. If it does, it will remove that key and print the corresponding value. If it doesn't, a value of `nil` is returned.

```

Reply:
I have taken this comment into consideration and have implemented it, thank you.